strapi/strapi
# feat(history): capture content types history

### What does it do?

- creates a history service
- creates a document middleware that automatically saves history when entries are created or updated

### Why is it needed?

To make history

### How to test it?

- Use a database explorer and look up the strapi_history_versions table
- Update an entry in the admin, you should see a new row in the table with all the columns filled
- Same thing for entry creation
- No other actions should create new rows
